submissions/Levi123/html-css-popup/index.html

```diff
@@ -0,0 +1,111 @@
+                            <div>Calendar</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/mail.png" alt = "mail" class = "imgMenu"/></a>
+                            <div>Mail</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/news.png" alt = "news" class = "imgMenu"/></a>
+                            <div>News</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/maps.png" alt = "maps" class = "imgMenu"/></a>
+                            <div>Maps</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/files.png" alt = "files" class = "imgMenu"/></a>
+                            <div>Files</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/contacts.png" alt = "contacts" class = "imgMenu"/></a>
+                            <div>Contacts</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/tanslation.png" alt = "translation" class = "imgMenu"/></a>
+                            <div>Translation</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/videos.png" alt = "videos" class = "imgMenu"/></a>
+                            <div>Videos</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/search.png" alt = "search" class = "imgMenu"/></a>
+                            <div>Search</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/photos.png" alt = "photos" class = "imgMenu"/></a>
+                            <div>Photots</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/apps.png" alt = "apps" class = "imgMenu"/></a>
+                            <div>Apps</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/documents.png" alt = "documents" class = "imgMenu"/></a>
+                            <div>Documents</div>
+                        </li>
+                    </ul>
+                    <ul class = "popUpMoreElements">
+                        <input type="checkbox" id="popUpMore">
+                        <label for="popUpMore" class="popUpBtnMore">More</label>
```

Comment:
More button should have `:hover` and `:focus` states

Reply:
Add hover and focus to more button